text-icu.cabal

```diff
@@ -82,12 +82,7 @@ library
       Data.Text.ICU.Spoof.Pure
       Data.Text.ICU.Text
   c-sources: cbits/text_icu.c
-  include-dirs: include
-  extra-libraries: icuuc
-  if os(mingw32)
-    extra-libraries: icuin icudt
-  else
-    extra-libraries: icui18n icudata
+  include-dirs: include, /usr/local/opt/icu4c/include
```

Comment:
We needed to add brew’s include path for icu4c so that @mdiep could build on his MacBook. Don’t ask me why it didn’t fail otherwise :sob: